packages/libs/error-stack/tests/snapshots/doc/fmt__preference_color.snap

```diff
@@ -0,0 +1,9 @@
+<b>invalid input parameter</b>
+<span style='color:#a00'>├</span><span style='color:#a00'>╴</span><span style='color:#a00'> </span><span style='color:#555'>src/fmt/color.rs:24:5</span>
```

Comment:
I have not noticed that before as the output is the same, but do we emit unneeded spans here, or is this due to the conversion to HTML-codes?

Reply:
Great catch. After looking at the code, we actually emit the ANSI codes multiple times, while in reality, there's no real change. It is considered poor etiquette. Fixed in [8f5918b](https://github.com/hashintel/hash/pull/1800/commits/8f5918bfa5cec7b666cf544ed74ac8663a6ce85d)